lib/api/controllers/remoteActions/swagger.js

```diff
@@ -82,9 +82,7 @@ generateSwagger = (routes, apiVersion) => {
       }
 
       while ((m = reg.exec(route.url)) !== null) {
-        if (m.index === reg.lastIndex) {
-          reg.lastIndex++;
-        }
+        reg.lastIndex++;
```

Comment:
again, no need to increment reg.lastIndex here, the cursor is already placed at the good index to continue the searh.